CounterStrike/src/dev/danablend/counterstrike/runnables/Bomb.java

```diff
@@ -1,72 +1,91 @@
 package dev.danablend.counterstrike.runnables;
 
-import java.text.DecimalFormat;
-
-import org.bukkit.Bukkit;
-import org.bukkit.ChatColor;
-import org.bukkit.Effect;
-import org.bukkit.GameMode;
-import org.bukkit.Location;
-import org.bukkit.Material;
-import org.bukkit.Sound;
-import org.bukkit.entity.Player;
-import org.bukkit.event.EventHandler;
-import org.bukkit.event.HandlerList;
-import org.bukkit.event.Listener;
-import org.bukkit.scheduler.BukkitRunnable;
-
 import com.gmail.filoghost.holographicdisplays.api.Hologram;
-import com.gmail.filoghost.holographicdisplays.api.HologramsAPI;
 import com.gmail.filoghost.holographicdisplays.api.line.TextLine;
-
 import dev.danablend.counterstrike.Config;
 import dev.danablend.counterstrike.CounterStrike;
+import dev.danablend.counterstrike.GameState;
 import dev.danablend.counterstrike.csplayer.CSPlayer;
 import dev.danablend.counterstrike.events.WeaponFireEvent;
 import dev.danablend.counterstrike.utils.PacketUtils;
+import org.bukkit.*;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.HandlerList;
+import org.bukkit.event.Listener;
+
+import java.text.DecimalFormat;
+import java.util.concurrent.atomic.AtomicInteger;
 
-public class Bomb extends BukkitRunnable {
+public class Bomb {
 
     public static Bomb bomb = null;
     public static boolean detonated = false;
-
+    private static boolean cleaning = false;
+    Object global;
+    Object bombTask;
+    Object defuseTask;

```

Comment:
access modifier is missing, should this be `protected`?
And also look at the comment in MyBukkit regarding `Object` type
```suggestion
    private Object global;
    private Object bombTask;
    private Object defuseTask;

```